Sources/SwiftSyntax/generated/TokenKind.swift

```diff
@@ -14,7 +14,6 @@
 
 /// Enumerates the kinds of tokens in the Swift language.
 public enum TokenKind: Hashable {
-  case eof
```

Comment:
I agree. Could you add a deprecated compatibility layer to SwiftSyntaxCompatibility.swift? We can’t achieve 100% source compatibility because you could switch exhaustively over `TokenKind` before and the rename of `eof` would break that  but the following should catch many cases.

```swift
extension TokenKind {
  @available(*, deprecated, renamed: "endOfFile")
  static var eof: TokenKind { .endOfFile }
}
```